bluebottle/members/models.py

```diff
@@ -185,6 +187,20 @@ class MemberPlatformSettings(BasePlatformSettings):
         )
     )
 
+    @property
+    def fiscal_year_start(self):
+        offset = self.fiscal_month_offset
+        month_start = (datetime(2000, 1, 1) + relativedelta(months=offset)).month
+        return (now() + relativedelta(months=offset)).replace(month=month_start, day=1, hour=0, second=0)
```

Comment:
This looks wrong. You take now, add the offset in months and then replace the month with the calculated month offset. Why not take the start of the year and set the month. 
I think this probably works, but it is not clear to me at all why.